core/data/init.go

```diff
@@ -28,6 +28,10 @@ import (
 	"github.com/edgexfoundry/edgex-go/support/logging-client"
 )
 
+const (
```

Comment:
This would be better served being added to core/data/const.go. That's where I put it in https://github.com/edgexfoundry/edgex-go/pull/17

Reply:
I agree